source/whats-new.txt

```diff
@@ -22,11 +22,19 @@ Overview
 
 Learn what's new in:
 
+* :ref:`Version 4.1 <version-4.1>`
 * :ref:`Version 4.0 <version-4.0>`
 * :ref:`Version 3.11 <version-3.11>`
 * :ref:`Version 3.10 <version-3.10>`
 * :ref:`Version 3.9 <version-3.9>`
 
+.. _version-4.1:
+
+What's New in 4.1
```

Comment:
To clarify, we have not yet released 4.1. We might want to consider delaying documenting the 4.1 version until it is released to avoid confusing users. If these docs are not made available yet to users immediately following PR merge, disregard this comment.

Reply:
We maintain a branch in each docs repo called `upcoming` where we keep changes for the next version. A user would need to consciously select `upcoming` in order to see the v4.1 docs.